index.html

```diff
@@ -456,6 +456,48 @@ <h3>Programming Diversity</h3>
     			$('.first-button').on('click', function () {
   				$('.animated-icon').toggleClass('open');
     			});
+
+// added code to resolve safari smooth scrolling error
```

Comment:
perhaps an approach here could be to add the issue number this code resolves? I am not familiar with the practice of adding a comment to note that code was added to resolve something specific but it seems useful :)